### ISSUE TITLE
Fan Management: Add Search to Fans list 

### ISSUE THREAD

**Issue by Tas4tari** [big-neon/bn-web#1672]
User should be able to search the Fans list for a specific user. https://develop.bigneon.com/admin/fans

**Comment by Anziskavds** [comment]
Tested develop on web and mobile web and a user is only able to search by one name at a time. User can't search both the first and last name. This occurs on mobile web and web. Also found on Mobile web after you typed in the name you wish to search for, it gives you no fans currently and there is no search bar to clear the bar and try to search with another name or maybe fix a typing error. 

Web: https://cl.ly/da58f4e7d101
Mobile web: https://cl.ly/ad13a8ace84e

**Comment by Timo614** [comment]
Added fuzzy searching for the names so first and last work which were missing on the API.

QA failed the issue for some bugs noticed: https://github.com/big-neon/bn-api/issues/1430#issuecomment-517581493

I noticed the issues appear to be UI related:
https://github.com/big-neon/bn-api/issues/1430#issuecomment-517738456

- The search box is using the page that the user is currently on but should be always reseting back to page 1. If you do a search from page 2 and there's only 1 result page it ends up showing a not found error at the moment.
- It may be worth adding some sort of indicator or overlay to indicate that the UI is rendering results. The really slow list like fans seems like things are broken as it takes a bit to render (just seemed like search pagination was broken and that it didn't reset the page when you remove queries, etc since it wasn't clear it was loading)
- It looks like if a component is rendering and someone clicks a different link the initial rendering continues to occur until it's ready and it renders. If you click a bunch of different pages for a slow loading organization fans list it renders one after another as they finish rendering.
